### ISSUE TITLE
not all static resources being served from static.login.anosrep.org

### ISSUE THREAD

**Issue by jrgm** [mozilla/persona#1911]
The js and css are served from static.login.anosrep.org, but the images and fonts are coming from login.anosrep.org. GH-1741 says "static.login.anosrep.org will be the location all resources are served from"

For example here are the requests I see when doing the following with stage:

Load beta.123done.org (123done content omitted):

```
GET 200 http://beta.123done.org/
GET 302 to: https://login.anosrep.org/include.js https://diresworb.org/include.js
GET 200 https://login.anosrep.org/include.js
GET 200 https://login.anosrep.org/communication_iframe
GET 200 https://static.login.anosrep.org/v/b7cb529baa/production/communication_iframe.js
GET 200 https://login.anosrep.org/wsapi/session_context
```

Click 'Sign in with Persona':

```
GET 200 https://login.anosrep.org/sign_in
GET 200 https://static.login.anosrep.org/v/e45f9f9309/production/dialog.css
GET 200 https://static.login.anosrep.org/v/efe04150e0/production/en_US/dialog.js
GET 200 https://login.anosrep.org/v/fb16875e7d/i/grain.png
GET 200 https://login.anosrep.org/v/72323e521e/i/persona-logo-transparent.png
GET 200 https://login.anosrep.org/v/8bf925a5b2/i/arrow_grey.png
GET 200 https://login.anosrep.org/v/d5eabde735/fonts/OpenSans-Regular.woff
GET 200 https://login.anosrep.org/v/28c393ca13/fonts/OpenSans-Light.woff
GET 200 https://login.anosrep.org/wsapi/session_context
GET 200 https://login.anosrep.org/favicon.ico
GET 304 https://login.anosrep.org/favicon.ico
GET 200 https://login.anosrep.org/v/9d9a8cdfd7/i/button-arrow.png
GET 200 https://login.anosrep.org/v/f577763f76/fonts/OpenSans-Bold.woff
```

Load http://diresworb.org (main site)

```
GET 301 to: https://diresworb.org/ http://diresworb.org/
GET 302 to: https://login.anosrep.org/ https://diresworb.org/
GET 200 https://login.anosrep.org/
GET 200 https://static.login.anosrep.org/v/fb5534092a/production/browserid.css
GET 200 https://static.login.anosrep.org/v/585956c401/production/en_US/browserid.js
GET 200 https://static.login.anosrep.org/v/ad5ad4541d/i/slit.png
GET 200 https://login.anosrep.org/v/d5eabde735/fonts/OpenSans-Regular.woff
GET 200 https://login.anosrep.org/v/28c393ca13/fonts/OpenSans-Light.woff
GET 200 https://login.anosrep.org/v/f577763f76/fonts/OpenSans-Bold.woff
GET 200 https://login.anosrep.org/v/4d0ad62cd5/i/marketplace-header.png
GET 200 https://login.anosrep.org/v/f3427f7dd5/i/persona-logo-wordmark.png
GET 200 https://login.anosrep.org/v/7021b96f44/i/badge.png
GET 200 https://login.anosrep.org/wsapi/session_context
GET 304 https://login.anosrep.org/favicon.ico
```

**Comment by lloyd** [comment]
Nice find!  Non-critical for this train, but we certainly should fix it!  

--lloyd

**Comment by seanmonstar** [comment]
From dev.123done.org, which takes me to login.dev.anosrep.org, static resources are served from static.dev.anosrep.org, with the exception of the favicon.

Is this a configuration thing per server, or just been fixed since the last stage was cut?

**Comment by lloyd** [comment]
Let's verify it's fixed on stage once we push train 06.07.2012, if it is not fixed, it must be a config issue, right?

Thanks 4 investigating, Sean.

--lloyd

**Comment by seanmonstar** [comment]
This seems to be a config issue, since stage has the 07.20.2012 train, and resources are still being pulled in from login, and not from static. Who can fix that sort of thing?

**Comment by lloyd** [comment]
this issue is fixed in dev.  @jrgm are you comfortable with me closing this and us verifying it once we get train-2012.08.17 onto the actual stage environment?

**Comment by zaach** [comment]
@lloyd what was the problem/fix?

**Comment by seanmonstar** [comment]
Pretty sure its not a code fix. Its been those way for a couple trains. I
believe its a configuration thing.

**Comment by jrgm** [comment]
So, it does appear to be fixed in dev. Closing, and we'll see if it's fixed in stage when we get there.

**Comment by seanmonstar** [comment]
@lloyd @gene1wood @jrgm Was this the bug you said managed to carry over to production? We closed it because it wasn't in dev.

**Comment by seanmonstar** [comment]
Yea, looking at https://login.persona.org, some of the static files are being served from login. Re-opening.

**Comment by seanmonstar** [comment]
CSS stylesheets are using https://login.persona.org hardcoded in them, which must be built during rpmbuild time. @ozten does this sound familiar?

**Comment by seanmonstar** [comment]
No, that's not it. i've found the exact location this is. The `cachify_prefix` option gets appended to the `public_static_url`, which if it doesn't exist, falls back to the `public_url`.

So the real action here is to set the `public_static_url` to `https://static.login.persona.org`.

**Comment by seanmonstar** [comment]
closed with #2552
